composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.4.8",
+        "php": ">=5.5.9",
         "symfony/event-dispatcher": "^2.3||^3.0",
         "symfony/options-resolver": "^2.3||^3.0",
-        "psr/http-message-implementation": "~1.0",
-        "php-http/adapter-implementation": "^0.1.0",
-        "php-http/discovery": "^0.1.1",
-        "php-http/message-decorator": "^0.1.0"
+        "php-http/client-implementation": "^1.0.0",
+        "php-http/discovery": "^0.6.0",
+        "php-http/message": "^0.2.1",
+        "puli/cli": "^1.0.0-beta8"
```

Comment:
so the composer plugin relies on the puli cli being either in the
vendors or installed globally?

what happens if it is not? will composer install explode, or is it
silently ignored and there are no resources? can discovery figure out
whether there are no resources because no adapter was installed or
because puli cli is missing? aka, can we properly report the problem to
the user?